frameworks/g-cloud-9/questions/services/publicSectorNetworksTypes.yml

```diff
@@ -15,13 +15,19 @@ type: checkboxes
 options:
   - label: Public Services Network (PSN)
     value: psn
+    filter_label: Public Services Network (PSN)
```

Comment:
Guess it doesn't matter now, but we don't really need to add filter_labels if they are the same as the existing label. Would be nice to only override if it there's a difference.